tests/suit/vsphere_test.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 const (
-	providerName          = "vsphere-provider"
+	vsphereProviderName   = "vsphere-provider"
+	ovirtProviderName     = "ovirt-provider"
```

Comment:
in vsphere_test?..

Reply:
done , also moved the shared constants to the tests_suit_test.go which is common for both.